vmdb/app/views/shared/summary/_textual_multilabel.html.haml

```diff
@@ -11,5 +11,9 @@
     - items[:values].each do |item|
       %tr
         - item.each do |value|
-          %td
-            = value
+          - if value.is_a?(Hash)
```

Comment:
There is a new hash option containing both 'title' and 'text'.
The old option of just passing a string with text value is still supported.